ui/cocoa/view.m

```diff
@@ -207,6 +221,32 @@ - (id)initWithFrame:(NSRect)frameRect
         [pauseLabel sizeToFit];
 
     }
+
+    CGEventMask mask = CGEventMaskBit(kCGEventKeyDown) | CGEventMaskBit(kCGEventKeyUp) | CGEventMaskBit(kCGEventFlagsChanged);
+    eventsTap = CGEventTapCreate(kCGHIDEventTap, kCGHeadInsertEventTap, kCGEventTapOptionDefault,
+                                 mask, (CGEventTapCallBack) handleTapEvent, self);
```

Comment:
I do not think the type casting of `handleTapEvent` is necessary. Please avoid type casting whenever possible, or the type system will fail to catch a real mistake.

Reply:
Very used to doing it on Gnome codebases and the Cocoa examples also had it, but I think you are right.

Reply:
Done.

Reply:
The fix is reverted.

Reply:
Sorry, I probably restored to a previous version of the commits when undoing the squash mistake.